gazebo_ros2_control/src/gazebo_system.cpp

```diff
@@ -608,9 +608,6 @@ hardware_interface::return_type GazeboSystem::write(
         this->dataPtr->sim_joints_[j]->SetVelocity(0, this->dataPtr->joint_velocity_cmd_[j]);
       } else if (this->dataPtr->joint_control_methods_[j] & EFFORT) { // NOLINT
         this->dataPtr->sim_joints_[j]->SetForce(0, this->dataPtr->joint_effort_cmd_[j]);
-      } else {
```

Comment:
According with this comment https://github.com/ros-controls/gazebo_ros2_control/pull/177#issuecomment-1584743086, I think this is required to avoid sagging

Reply:
@gwalck @AndyZe how can we support passive joints, if your robots require sending zero velocity?

Reply:
fixed with #237